pkg/filebuffer/filebuffer.go

```diff
@@ -113,6 +120,11 @@ func (fb *FileBuffer) Bytes() []byte {
 }
 
 func (fb *FileBuffer) Write(p []byte) (n int, err error) {
+	fb.mu.Lock()
+	defer fb.mu.Unlock()
+
+	fb.sourceMap = nil
```

Comment:
I suspect this isn't necessary, but added it out of an abundance of caution since I wasn't sure why `SourceMap` was changed to stop caching the `sourceMap` in the first place.